### PR TITLE
Enable encryption of notes and properties in .level files

### PR DESCRIPTION
**Pull request by davidsbailey**
### Background

implements [LP-168](https://codedotorg.atlassian.net/browse/LP-168)

### Description

The work item asks to encrypt the entire .level file. This solution encrypts only the `notes` and `properties` fields, because only those appear to contain sensitive data, and it's nice for engineers to be able to see other pertinent details such as level type and audit log without having to fire up levelbuilder.

### Future work

The short_instructions and long_instructions also need to be encrypted (see [LP-203](https://codedotorg.atlassian.net/browse/LP-203)).

### CODE REVIEW

File: dashboard/config/scripts/allthethings.script [hunk 1]
```diff
@@ -244,6 +245,3 @@ level 'allthethings_fish_tank'
 stage 'Dance Lab'
 level 'allthethings HOC Dance 1'
 level 'allthethings HOC Dance 10 Measures'
-
-stage 'Mini Rubric'
-assessment 'allthethings-mini-rubric'
```
Comment (islemaster) [inline dashboard/config/scripts/allthethings.script]: Intentionally removing the mini-rubric stage?

**Review by islemaster — APPROVED** [review]
:+1:

File: bin/i18n/sync-codeorg-in.rb [hunk 2]
```diff
@@ -111,6 +111,7 @@ def localize_level_content
 
       # Properties
       config = JSON.parse(level_xml.xpath('//../config').first.text)
+      next unless config["properties"]
```
Comment (davidsbailey) [inline bin/i18n/sync-codeorg-in.rb:114]: per convo with @Hamms

**Comment by davidsbailey** [comment]
Regarding the future work and summarizing a conversation with @Hamms  -- when looking for instructions, translations sync-in looks at the level XML directly and then pulls out any `short_instructions` or `long_instructions` it finds from the `properties`. because it parses XML directly rather than using load_level_xml, it won't find any `properties` for encrypted levels because these are now stored in `encrypted_properties` for encrypted levels. So, no additional work is needed to prevent instructions from being hidden for encrypted levels (although one line was needed not to break the sync-in ruby script).

File: dashboard/config/scripts/allthethings.script (hunk 1 above)
Comment (davidsbailey) [reply dashboard/config/scripts/allthethings.script]: oops, no! fixed.

**Pull request merged**